recipes/expat/expat.inc

```diff
@@ -6,6 +6,9 @@ RECIPE_TYPES = "machine native sdk"
 
 inherit autotools make-vpath
 
+#Only used for Aarch64
+AUTOCONFDIR = "/conftools"
```

Comment:
In core it's only read for aarch64 :-) So i guess i could add :TARGET_ARCH_aarch64, but is allready there in core... We wait for @esbens view of this before i do any more work on this